trace/Span.md

```diff
@@ -1,8 +1,11 @@
-# Span API
+# Span
 
-This class represents a Span. A span contains a SpanContext and  allows users to record 
-tracing events based on the data model defined [here](https://github.com/census-instrumentation/opencensus-proto/blob/master/trace/trace.proto).
-All Span implementations MUST not allow users to read the recorded data, 
+Span represents a single operation within a trace. Spans can be nested to form a trace tree. 
+Often, a trace contains a root span that describes the end-to-end latency and, optionally, one or
+more sub-spans for its sub-operations.
+ 
+A span contains a SpanContext and allows users to record tracing events based on the data model 
```

Comment:
Isn't span context also the identifier. We should focus on that responsibility before mentioning anything about recording tracing events.

Reply:
It would be good to clarify that SpanContext and Context are completely differently and pretty much unrelated, and SpanContext is really more like SpanHandle or SpanName or SpanControlBlock or SpanIdentifier (except span_id is already used).  Ok, I can't think of a good name either.